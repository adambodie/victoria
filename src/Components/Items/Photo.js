import React from 'react';

const Photo = ({id, title}) => (
	<div>
		<img src={`https://victoria.bodiewebdesign.com/Images/${id}_o.jpg`} alt={title}/>
		<h3>{title}</h3>
	</div>
);

export default Photo;
