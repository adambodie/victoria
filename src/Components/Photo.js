import React from 'react';

const Photo = props => (
  <div>
        <img src={`https://farm${props.farm}.staticflickr.com/${props.server}/${props.id}_${props.secret}_z.jpg`} alt={props.title}/>
        <h3 style={{color: props.color}}>{props.title}</h3>
        <h4 style={{color: props.color}}>{props.tags}</h4>
        <h6 style={{color: props.color}}>{props.index} of {props.length}</h6>
  </div>
);

export default Photo;
