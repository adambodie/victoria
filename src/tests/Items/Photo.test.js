import React from 'react';
import renderer from 'react-test-renderer';
import Photo from '../../Components/Items/Photo';

it('renders a fact', () => {
  const tree = renderer.create(<Photo id="25" title="Victoria"/>).toJSON();
  expect(tree).toMatchSnapshot();
});
