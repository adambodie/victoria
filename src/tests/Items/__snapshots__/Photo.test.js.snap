// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a fact 1`] = `
<div>
  <img
    alt="Victoria"
    src="..//assets/photos/25_o.jpg"
  />
  <h3>
    Victoria
  </h3>
</div>
`;
